xbmc/dbwrappers/sqlitedataset.h

```diff
@@ -116,8 +116,7 @@ class SqliteDataset : public Dataset
 
 public:
   /* constructor */
-  SqliteDataset();
-  explicit SqliteDataset(SqliteDatabase* newDb);
+  using Dataset::Dataset;
```

Comment:
"using" in a header? doesn't build without but doesn't seem right, this fixes the build
```suggestion
  SqliteDataset(SqliteDatabase* newDb) : Dataset(newDb) {}
```
mysql is the same way.

Reply:
> "using" in a header?

 Fine in this context. => https://stackoverflow.com/questions/57105400/using-declaration-for-base-class-constructors
 
 Avoids exactly writing the purely "proxying" ctor you suggest.

Reply:
learned something today! too many uses and tricks with using...